src/main/java/kostka/moviecatalog/controller/MovieDetailController.java

```diff
@@ -25,7 +25,7 @@
     public static final String IS_USER_ADULT_ATTR = "isUserAdult";
     public static final String STATUS_ATTR = "status";
     private static final String MOVIE_DETAIL_VIEW = "detail";
-    private static final String REDIRECT_MOVIE_DETAIL_VIEW = "redirect:/movies/detail/";
+    public static final String REDIRECT_MOVIE_DETAIL_VIEW = "redirect:/movies/detail/";
```

Comment:
OPTIONAL: Wrong order of properties. Maybe you can add the check to the checkstyle if it is not there: https://checkstyle.sourceforge.io/config_coding.html#DeclarationOrder ;)

Reply:
will be sorted out in separate MR, thanks for tip!